strings/strings.c

```diff
@@ -9,7 +9,16 @@
 */
 int string_length(char s[])
 {
-
+    // \0
+    int count = 0;
+    // printf("s => %c \n", s[26]);
+    for (int i = 0; ; i++){
+        if(s[i] == '\0'){
```

Comment:
why not throw this break condition in the center argument of your for loop.
```
{
    // \0
    int count = 0;
    for (; s[count] != '\0';){
        count++;
    };
    return count;
}
```

Reply:
Agreed, we discussed that in my peer code review this morning.